container-image/examples/openshift/helm-wildfly.yaml

```diff
@@ -0,0 +1,11 @@
+build:
+  uri: https://github.com/wildfly/quickstart.git
+  ref: 35.0.0.Final
+  contextDir: helloworld
+deploy:
+  replicas: 1
+  env:
+    - name: ADMIN_USERNAME
```

Comment:
That'd be better to have a `secret` holding the admin credentials and refer to the secret from both Helm charts with `deploy.envFrom`

Another UX improvement could be to have a Helm chart that would deploy all the resources (Quickstart, chatbot, Ollama)